pkgs/top-level/all-packages.nix

```diff
@@ -132,6 +132,8 @@ with pkgs;
 
   deadcode = callPackage ../development/tools/deadcode { };
 
+  tychus = callPackage ../development/tools/tychus { };
```

Comment:
Should be relocated further down between `tweak` and `uhd` to respect a (vague) alphabetical order in the `### DEVELOPMENT / TOOLS` section